content/blog/2018-09-04-birds-taxo-traits.md

```diff
@@ -0,0 +1,559 @@
+---
+title: "What are these birds? Complement occurrence data with taxonomy and traits information"
+slug: birds-taxo-traits
+authors:
+  - name: Maëlle Salmon
+    url: https://masalmon.eu/
+date: 2018-09-04
+preface: The blog post series corresponds to the material for a talk Maëlle will give at the [Animal Movement Analysis summer school in Radolfzell, Germany on September the 12th](http://animove.org/animove-2019-evening-keynotes/), in a Max Planck Institute of Ornithology.
+tags:
+- auk
+- taxize
+- traits
+- taxonomy
+- rphylopic
+- gifski
+- birder
+output: 
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+Thanks to the [second post of the series where we obtained data from
+eBird](https://ropensci.org/blog/2018/08/21/birds-radolfzell/) we know
+what birds were observed in the county of Constance. Now, not all
+species’ names mean a lot to me, and even if they did, there are a lot
+of them. In this post, we shall use rOpenSci’s packages accessing
+taxonomy and trait data in order to summarize the birds’ population of
```

Comment:
phrase "to summarize the birds’ population of the county" is unclear. Do you mean "summarize the types of birds that populate the county"?

Reply:
I made it "in order to summarize some characteristics of the birds' population"